### ISSUE TITLE
Any plans to strong-name-sign?

### ISSUE THREAD

**Issue by smoeller-liveid** [ststeiger/PdfSharpNetStandard#3]
Hello,
I like this project very much, but can only use strong-name-signed dependencies.
Is there any plan to sign the nuget version?

**Comment by ststeiger** [comment]
No, because then I can't update anymore without changing the references.
